testplan/testing/multitest/driver/sqlite.py

```diff
@@ -24,6 +24,7 @@ def get_options(cls):
         """
         return {
             "db_name": str,
+            ConfigOption("db_path", default=None): str,
```

Comment:
We need docstring for new config options.

Reply:
Yep :)